csharp/src/Drivers/Databricks/DatabricksConnection.cs

```diff
@@ -145,6 +169,22 @@ protected override HttpMessageHandler CreateHttpHandler()
             return baseHandler;
         }
 
+        protected internal override bool AreResultsAvailableDirectly() => _enableDirectResults;
```

Comment:
Another good place for a property:
```suggestion
        protected internal override bool AreResultsAvailableDirectly => _enableDirectResults;
```
Sorry; I missed this in my previous review.

Reply:
This is overriding AreResultsAvailableDirectly in HiveServer2Connection/SparkConnection, where it is a function. Should I update the field in both parent classes to be a property as well?

Reply:
Yes, please!